inductiva/utils/grids.py

```diff
@@ -0,0 +1,43 @@
+"""Utility functions to manipulate grid data."""
+import typing
+
+import scipy
+import numpy as np
+
+
+def get_meshgrid(x_range: typing.Sequence[float],
+                 y_range: typing.Sequence[float], x_num: int, y_num: int):
+
+    x_grid = np.linspace(*x_range, num=x_num)
+    y_grid = np.linspace(*y_range, num=y_num)
+
+    x_grid, y_grid = np.meshgrid(x_grid, y_grid, indexing="ij")
+
+    return x_grid, y_grid
+
+
+def interpolate_between_grids(x_num: int, y_num: int, z_array: np.ndarray):
```

Comment:
much cleaner. But the method name is not very friendly.
Why not 

```
reshape/rescale_map(z_array, target_x_num, target_y_num)
```

interpolation is the *strategy* we are using for this reshaping. It could be clipping, it could be mirroring, etc What we really want is the reshaping/rescaling. The method name should reflect that.